src/planwise/client/db.cljs

```diff
@@ -28,7 +28,10 @@
 
    ;; Projects
    :projects
-   {:creating? false}
+   {:creating? false
+    :creating-waiting? false
```

Comment:
`creating?` and `creating-waiting?` are actually representing a `creating-state` with 3 possible values. How should this be handled? Just symbols, or is there something closer to enums that can be used for this?